src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separ = ',') {
+    let result = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const element = this[i];
```

Comment:
Why we need this const?

Reply:
'element' sounds better :grin:
okay I'll change, my bad...